app/livechat/server/api/v1/room.js

```diff
@@ -103,7 +103,9 @@ API.v1.addRoute('livechat/room.transfer', {
 			// update visited page history to not expire
 			Messages.keepHistoryForToken(token);
 
-			if (!Promise.await(Livechat.transfer(room, guest, { roomId: rid, departmentId: department }))) {
+			const transferedBy = { _id: room.servedBy._id, username: room.servedBy.username };
```

Comment:
Since the `transferedBy`  information is already passed to `Livechat.transfer` through the `room` object, I think we don't need to change he signature of the method `Livechat.transfer`, we can get this information inside the method.
What do you think?